recursion/MainClass.java

```diff
@@ -51,6 +51,6 @@ static int fastPow(int a, int b) {
 	static int path(int n, int m) {
 		if(n == 1 || m == 1) return 1;
 		
-		return path(n, m-1) + path(m, n-1);
+		return path(n, m-1) + path(m, n-1);//no. of ways to traverse in n*m grid from top right corner to bottom left corner
```

Comment:
You should move this comment from the right side of the code to above the code line. Also, this comment looks like it should be a doc comment for our function path(). So it's better to make this comment as a JavaDoc function comment.

Reply:
Done bhaiya!!

Reply:
I think it's still not done correctly, @Shaikh-Nabeel can you help please😄.